test/src/test/java/hudson/cli/SetBuildDescriptionCommandTest.java

```diff
@@ -145,4 +148,14 @@
         assertThat(result.stderr(), containsString("ERROR: No such build #2"));
     }
 
+    //TODO: determine if this should be pulled out into JenkinsRule or something
```

Comment:
The XShell plugin provides something like this.  It takes a command line and attempts to map it to Windows or Linux as appropriate.